modules/cotton-cauldron/src/main/resources/fabric.mod.json

```diff
@@ -23,10 +23,11 @@
   "depends": {
     "fabricloader": ">=0.4.0",
     "cotton-commons": "*",
-    "libcd": ">=2.0.0"
+    "libcd": ">=3.0.0",
+    "libdp": ">=1.0.0"
```

Comment:
I don't think this actually hard-deps on libcd *or* libdp anymore. I'll take a look through and see, and if it does then I'll fix it myself.